src/components/jsmd-editor.jsx

```diff
@@ -138,14 +152,15 @@ class JsmdEditorUnconnected extends React.Component {
     //   },
     // )
 
-    // FIXME: should set cursor position in redux store (see: https://github.com/iodide-project/iodide/issues/1568)
+    const { editorCursorLine, editorCursorChar } = this.props;
     return (
       <ReactCodeMirror
         editorDidMount={this.storeEditorInstance}
-        cursor={{ line: 1, ch: 1 }}
```

Comment:
the schema sets these values to 0 by default. does this matter? Should it have been 0 originally?

Reply:
it should have been 0 originally